custom_components/thermal_comfort/translations/en.json

```diff
@@ -0,0 +1,43 @@
+{
+  "options": {
+    "step": {
+      "init": {
+        "title": "Thermal comfort settings",
+        "data": {
+          "temperature_sensor": "Temperature sensor",
+          "humidity_sensor": "Humidity sensor",
+          "poll": "Should we pool?",
```

Comment:
Typo

Reply:
Fixed. Was not familiar with this option meaning. 
Version from readme is used now.